README.md

```diff
@@ -177,16 +178,49 @@ Note that if you are running `extend_ca_cert` locally on the primary Puppet serv
 bolt plan run ca_extend::extend_ca_cert --targets local://$(hostname -f) --run-as root
 ```
 
+### ca_extend::upload_ca_cert Plan
+
+You can use this plan with `cert` parameter to specify the location of the updated CA cert and distribute it to the nodes specified in the `targets` parameter
+
 ```bash
 bolt plan run ca_extend::upload_ca_cert cert=<path_to_cert> --targets <TargetSpec>
 ```
 
+### ca_extend::check_ca_expiry Task
+
+You can use this task to check the ca cert expiry on the `primary` mainly but you can also use it to check that a remote CA cert has been updated after using any means to update an expired CA cert on an agent.
+
 ```bash
 bolt task run ca_extend::check_ca_expiry --targets <TargetSpec>
 ```
 
+### ca_extend::check_agent_expiry Task
+
+You can use this task to categorise all PE certs in a PE environment as part of a valid or expiring section based on a customasable date in the future (default 3 months from now). This task runs against a `primary` server and checks all cer `/etc/puppetlabs/puppet/ssl/ca/signed` as the single source of truth for the PE environment and splits the certs between a valid section or expiring section. 
+
 ```bash
-bolt task run ca_extend::check_agent_expiry --targets <TargetSpec>
+bolt task run ca_extend::check_agent_expiry --targets local://$(hostname -f)
```

Comment:
If the target can only be localhost, would it make sense to simplify this to "--targets localhost"

Reply:
The readme mentions in the Usage section:
"Note that if you are running extend_ca_cert locally on the primary Puppet server, you can avoid potential Bolt transport issues by specifying --targets local://$(hostname -f)"

So, I went with that reference. Happy to update, if necessary

Reply:
@jordi if the docs say it lets go with that! id just not seen it before

Reply:
So per discussion local:// forces the local transport incase hosts are screwed up, but (hostname - f) introduces variability, i think we should just go local://localhost

Reply:
The `local://` part is a specific protocol that avoids trying to `ssh` into the box we're currently on, which has caused issues before.  I'm not sure if `$(hostname -f)` vs `localhost` would make much of a difference, but I think I'd prefer the `hostname` approach.  Someone could have a weird IPv6 `localhost` configuration or something, and the `-f` flag should be portable.  Except Solaris, let's not speak of that...

Reply:
Asked about this in #bolt, and it doesn't actually matter what the rest of the URI has, as long as the protocol is `local://`.